README.md

```diff
@@ -1,70 +1,26 @@
-# Getting Started with Create React App
+# Mapanimal
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
 
-## Available Scripts
+## You can clone this repository following way:
```

Comment:
You can clone this repository following way => You can clone this repository in following way: